### DISCUSSION TITLE
Radio Button in Form

### DISCUSSION THREAD

**Discussion by Louis-de-Bontin** [py-pdf/pypdf#1360] [Q&A]
I need to update the value of a radio button when filling a form. 
I can't find any documentation, or any relevant discussion about it. 
I can update text fields or check boxes, but PyPDF2 doesn't seem to recognize the radio buttons.

Here is my code :

```python
from PyPDF2 import PdfReader, PdfWriter

reader = PdfReader("./myfile_path/myfile.pdf")
writer = PdfWriter()
page = reader.pages[0]
writer.add_page(page)

data = {"field1": "some text", "radio_button_group_name": "/value"}
for field, value in data.items():
    writer.update_page_form_field_values(writer.pages[0], {field: value})

with open("out_file.pdf", "wb") as f:
    writer.write(f)
```

The text field is updated, but the radio button is ignored, am I missing something ?

**Comment by MartinThoma** [comment]
PyPDF2 still has several issues with froms (see https://github.com/py-pdf/PyPDF2/labels/workflow-forms, especially https://github.com/py-pdf/PyPDF2/issues/1334).

This discussion seems to be a duplicate of https://github.com/py-pdf/PyPDF2/discussions/1283 . The problem is that I don't know how to do this. Hence I cannot help.

**Reply by fernandesjeevan** [reply]
is this issue solved? is there a way to handle update radiobutton values using pypdf2 or pypdf

**Reply by stefan6419846** [reply]
@fernandesjeevan I recommend you to open a new discussion in the Q&A section with your current code for *pypdf* (PyPDF2 is not supported anymore) and file. This makes it easier to analyze and help you with it.
